### ISSUE TITLE
Images Not Appearing in One Feed

### ISSUE THREAD

**Issue by PaulHarder2** [QuiteRSS/quiterss#1106]
When I read the Kevin Drum Feed at Mother Jones (http://feeds.feedburner.com/KevinDrum) and either double-click the item title or right-click it and choose "Open in Browser", I get a fully styled web page view -- except that none of the images in the page show up.  To read this feed properly, I have to open it in an external browser.  Can you fix this?

**Comment by genodeftest** [comment]
In *Tools*→ *Options* → *Browser* → *General*, is displaying images enabled?

**Comment by PaulHarder2** [comment]
Yes, I do see image display enabled.  However, it has been quite some time
since I submitted this bug report, and it appears that the bug has been
fixed. I had all but forgotten about submitting the report.  A few weeks
later, the internal browser again started displaying images for posts in
that feed.

__________________________
*Paul H. Harder II, Ph.D., AMS*
Houston, TX
281-793-7743
